behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -369,30 +402,21 @@ public function assertRegionElementHasClass($selector, $region, $class) {
   }
 
   /**
-   * @When I wait for messagebox to close
```

Comment:
Yup - noticed that in there when I was looking at your code.  I don't even know why its there cause I just use "Then I break" when I'm debugging (or just run in debug mode).